app/Models/Activite.php

```diff
@@ -72,4 +74,17 @@ public function id_apostolats(){
 
         return $this->id_apostolats_concernes;
     }
+
+    public function concerned(){
+        $conerned = "";
```

Comment:
there is a typo here I think, either remove the line or fix the typo if initialization is required.